### ISSUE TITLE
Test 1 unittest not run

### ISSUE THREAD

**Issue by kebp** [openbabel/openbabel#2161]
- [x] I believe this to be a bug with Open Babel
- [ ] This is a feature request

## Environment Information
Open Babel version: 3.0.0
Operating system and version: CentOS 8.1.1911

## Expected Behavior
After building OpenBabel I expected ctest to run all the tests and for them all to pass.

## Actual Behavior
The 1st test (unittest) is skipped (Not Run).  This appears to be because ctest expects the unittest executable to be in build/tests whereas it's actually in build/bin. After copying unittest to build/test all tests are run and pass.

**Comment by ghutchis** [comment]
Can you show me your test output before and after? With OB3, there's no 'unittest' - everything is done through a test runner program.

**Comment by ghutchis** [comment]
Mine most definitely shows the unit cell tests:
```
        Start 180: test_unitcell_1
180/199 Test #180: test_unitcell_1 ..................   Passed    0.01 sec
```

**Comment by kebp** [comment]
Before copying unit test to build/test I get:
        Start   1: unittest
```
Could not find executable /home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/unittest
Looked in the following places:
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Release/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Release/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Debug/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Debug/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/MinSizeRel/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/MinSizeRel/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/RelWithDebInfo/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/RelWithDebInfo/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Deployment/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Deployment/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Development/unittest
/home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Development/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Release/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Release/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Debug/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Debug/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/MinSizeRel/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/MinSizeRel/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/RelWithDebInfo/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/RelWithDebInfo/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Deployment/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Deployment/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Development/unittest
home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/Development/unittest
Unable to find executable: /home/kevin2/builds/openbabel/openbabel-3.0.0/build/test/unittest
  1/193 Test   #1: unittest .........................***Not Run   0.00 sec

```
After copying I get:
```
        Start   1: unittest
  1/193 Test   #1: unittest .........................   Passed    0.09 sec
```
I guess one potential difference could be that I'm aiming to do a local build in my home directory. I build with:
```
cmake .. -DCMAKE_INSTALL_PREFIX=/home/kevin/software/openbabel/3.0.0
-DPYTHON_BINDINGS=ON -DENABLE_TESTS=ON -DPYTHON_EXECUTABLE=/usr/bin/python3
-DRUN_SWIG=ON -DBOOST_ROOT=/usr/include/boost169 -DBOOST_LIBRARYDIR=/lib64/boost169/
```
Not sure if that would make a difference

**Comment by ghutchis** [comment]
See, that's my point. I don't have anything even close to that. Can you zip your source directory and post it for me? There is literally nothing in `test/CMakeFiles.txt` that would match `unittest`

**Comment by kebp** [comment]
It looks to me as if it's pulled in as part of maeparser.  Unittest is added as a target in external/maeparser-1.1/CMakeLists.txt when ENABLE_TESTS is on, I can see it built during make, and it's included in CTestTestfile.cmake in  my build directory.

Build log (make.log) and bziped src directory attached.

[src.zip](https://github.com/openbabel/openbabel/files/4523846/src.zip)
[make.log](https://github.com/openbabel/openbabel/files/4523832/make.log)

**Comment by kebp** [comment]
This appears to be fixed in 3.1.1
